action/action-simulator/src/main/java/com/powsybl/action/simulator/tools/ActionSimulatorTool.java

```diff
@@ -74,32 +88,43 @@ public String getDescription() {
             @Override
             public Options getOptions() {
                 Options options = new Options();
-                options.addOption(Option.builder().longOpt("case-file")
+                options.addOption(Option.builder().longOpt(ActionSimulatorTool.CASE_FILE)
                         .desc("the case path")
                         .hasArg()
                         .argName("FILE")
                         .required()
                         .build());
-                options.addOption(Option.builder().longOpt("dsl-file")
+                options.addOption(Option.builder().longOpt(ActionSimulatorTool.DSL_FILE)
                         .desc("the Groovy DSL path")
                         .hasArg()
                         .argName("FILE")
                         .required()
                         .build());
-                options.addOption(Option.builder().longOpt("contingencies")
+                options.addOption(Option.builder().longOpt(ActionSimulatorTool.CONTINGENCIES)
                         .desc("contingencies to test")
                         .hasArg()
                         .argName("CONTINGENCY1,CONTINGENCY2,...")
                         .build());
-                options.addOption(Option.builder().longOpt("verbose")
+                options.addOption(Option.builder().longOpt(ActionSimulatorTool.VERBOSE)
                         .desc("verbose mode")
                         .required(false)
                         .build());
-                options.addOption(Option.builder().longOpt("output-csv")
+                options.addOption(Option.builder().longOpt(ActionSimulatorTool.OUTPUT_CVS)
                         .desc("the CSV output path")
                         .hasArg()
                         .argName("FILE")
                         .build());
+                options.addOption(Option.builder().longOpt(ActionSimulatorTool.OUTPUT_CASE_FOLDER)
+                        .desc("output case folder path")
+                        .required()
```

Comment:
We should make it optional instead of required:
in case it is not defined, just don't export the cases (don't create the case exporter).